### PR TITLE
Remove deprecated parameter hold_trajectory_duration

### PR DESCRIPTION
**Pull request by AndyZe**

### CODE REVIEW

**Review by bmagyar — APPROVED** [review]
Thanks for this.
Even though I'm approving now the CI may still fail if we have tests directly aimed at this. If any, those will need to be removed.

**Review by bmagyar — CHANGES_REQUESTED** [review]
This change also removes the parameter we'd like to keep :)

File: joint_trajectory_controller/include/joint_trajectory_controller/joint_trajectory_controller_impl.h [hunk 1]
```diff
@@ -229,6 +229,7 @@ bool JointTrajectoryController<SegmentImpl, HardwareInterface>::init(HardwareInt
 
   // Stop trajectory duration
   stop_trajectory_duration_ = 0.0;
+  controller_nh_.getParam("stop_trajectory_duration", stop_trajectory_duration_);
```
Comment (mathias-luedtke) [inline joint_trajectory_controller/include/joint_trajectory_controller/joint_trajectory_controller_impl.h:232]: For semantic reasons I'd prefer
```
stop_trajectory_duration_ =  controller_nh_.param("stop_trajectory_duration", 0.0);
```

**Comment by mathias-luedtke** [comment]
Tests pass again, `hold_trajectory_duration` is not used elsewhere (in our repos).
https://github.com/ros-industrial/ur_modern_driver still has the old parameter name, but commented out.

**Review by bmagyar — APPROVED** [review]
Other than the point @ipa-mdl raised I'm happy with the PR. Please feel free to merge once done

File: joint_trajectory_controller/include/joint_trajectory_controller/joint_trajectory_controller_impl.h (hunk 1 above)
Comment (mathias-luedtke) [reply joint_trajectory_controller/include/joint_trajectory_controller/joint_trajectory_controller_impl.h:232]: There is a copy&paste bug in the new commit. I rolled back to the original version again.
In addition the other parameters are retrieved the same way, so let's keep it like this for now.

**Pull request merged**